src/openrct2/actions/MazeSetTrackAction.h

```diff
@@ -12,7 +12,7 @@
 #include "GameAction.h"
 // clang-format off
 /** rct2: 0x00993CE9 */
-static constexpr const uint8_t byte_993CE9[] = {
+static constexpr const uint8_t Byte993CE9[] = {
```

Comment:
are these used by anything other than maze set track action? Should move them into the .cpp file if its the only user.